_tour/implicit-conversions.md

```diff
@@ -38,6 +38,60 @@ In the second case, a conversion `c` is searched for, which is applicable to `e`
 
 An example is to compare two strings `"foo" < "bar"`. In this case, `String` has no member `<`, so the implicit conversion `Predef.augmentString("foo") < "bar"` is inserted. (`scala.Predef` is automatically imported into all Scala programs.)
 
+### How are implicit conversions brought into scope? ###
+
+{% tabs implicit-conversion-scope class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+In Scala 2, an implicit conversion is brought into scope by importing from the object that defined it, (e.g. `Conversions` in this case).
+
+```scala mdoc
+case class Student(name: String) {
+  def printName: Unit = println(name)
+}
+  
+object Conversions {
+  implicit def fromStringToStudent(name: String): Student = Student(name)
+}
+
+import Conversions._
+object Usage:
+  def main(args: Array[String]) = "Reginald".printName
+```
+{% endtab %}
+{% tab 'Scala 3' %}
+In Scala 3, an implicit conversion is brought into scope by either importing `given` or the named conversion from the object that defined it, (e.g. `Conversions` in this case). 
+
+Note that as of Scala 3, implicit conversions cannot be brought into scope anymore by means of a wildcard import (`*`). 
+
+Given the example:
+
+```scala mdoc
+case class Student(name: String):
+  def printName: Unit = println(name)
+object Student:
+  given Conversion[Student, Int] = _.name.length
+  
+object Conversions:
+  given fromStringToStudent: Conversion[String, Student] = Student(_)
+```
+
+The following imports would bring the `Conversion[String, Student]` into scope:
+  - `import Conversions.given`
+  - `import Conversions.{given[String, Student]}`
+  - `import Conversions.fromStringToStudent`
+
+If the implicit conversion is in the companion object of the argument type, (e.g. `Student` in this case), then no import is necessary.
```

Comment:
this also applies to Scala 2

Reply:
Thanks, I added it to the Scala 2 example as well.